src/lib/components/Cytoscape.react.js

```diff
@@ -266,10 +292,20 @@ class Cytoscape extends Component {
             sendSelectedEdgesData();
         });
 
-        cy.on('add remove', () => {
+        cy.on('add remove ', () => {
```

Comment:
?

Reply:
Hmm seems like I left that one behind...